x-pack/test/spaces_api_integration/common/suites/update_objects_spaces.ts

```diff
@@ -84,8 +93,25 @@ export function updateObjectsSpacesTestSuiteFactory(esArchiver: any, supertest:
             expect(result.type).to.eql(TYPE);
             expect(result.id).to.eql(id);
             expect(result.spaces.sort()).to.eql(expectedSpaces.sort());
+
+            if (expectAliasDifference !== undefined) {
+              // if we deleted an object that had an alias pointing to it, the alias should have been deleted as well
+              await es.indices.refresh({ index: '.kibana' }); // alias deletion uses refresh: false, so we need to manually refresh the index before searching
```

Comment:
NIT/optional: I think we could refresh only once if `objects.find((obj) => obj.expectAliasDifference !== undefined)`